rskj-core/src/test/java/org/ethereum/rpc/Web3ImplLogsTest.java

```diff
@@ -122,7 +123,7 @@ class Web3ImplLogsTest {
     private static final String ONE_TOPIC = "0000000000000000000000000000000000000000000000000000000000000001";
     private static final String INCREMENT_METHOD_SIGNATURE = "371303c0";
     private static final String GET_VALUE_METHOD_SIGNATURE = "20965255";
-    private static final String TRACKED_TEST_BLOCK_HASH = "0x8eac2053d453195bfede97c77ee215b64bf47c8bdcecc75c67a8aa6f7c66f173";
+    private static final String TRACKED_TEST_BLOCK_HASH = "0x93cdec69f2708ff7d744d607d59d2f59fc059579f7476e487206bc72ef2fd898";
```

Comment:
Because with the change in cost, the transaction hash also change. If I don't change this, the test will fail because we are searching for the wrong hash.